.github/workflows/ci-baseline.yml

```diff
@@ -32,36 +32,38 @@ jobs:
       fail-fast: false
       matrix:
         image:
-          - ubuntu-20.04-gnu_9-eigen_3.3.7-gtest
-          - ubuntu-20.04-clang_9-eigen_3.3.7-gtest
-          - fedora-34-gnu_11-eigen_3.3.7-gtest
-          - fedora-34-clang_12-eigen_3.3.7-gtest
-          #- intel_oneapi-eigen_3.3.7-gtest
+          - fedora-clang
+          - fedora-gnu
+          - ubuntu-clang
+          - ubuntu-gnu
+          #- intel-oneapi
```

Comment:
if all these are renamed this way, how do we know what the compiler is? we would have to go and check the CI run itself. is there a way we can keep the compiler version/name inthe name?